src/System.Management.Automation/engine/parser/TypeInferenceVisitor.cs

```diff
@@ -157,9 +157,9 @@ public bool TryGetRepresentativeTypeNameFromExpressionSafeEval(ExpressionAst exp
             {
                 return false;
             }
-            object value;
+
             return expression != null &&
-                   SafeExprEvaluator.TrySafeEval(expression, ExecutionContext, out value) &&
+                   SafeExprEvaluator.TrySafeEval(expression, ExecutionContext, out var value) &&
```

Comment:
The PR contains so many style changes that makes it unreasonably large.
Let's leave only significant changes - it will greatly facilitate and speed up the code viewing. Please move style changes in another PR.